src/OneExplorer.ts

```diff
@@ -172,29 +196,123 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
     }
   }
 
+  // TODO(dayo) extract file-relative functions as another module
+  private parseInputPath = (configPath: string, modelPath: string): string|undefined => {
+    const config = readIni(configPath);
+    const ext = path.extname(modelPath).slice(1);
+
+    if (ext === undefined || config === null) {
+      return undefined;
+    }
+
+    return config[`one-import-${ext}` as keyof typeof config] ?
+        config[`one-import-${ext}` as keyof typeof config]['input_path'] :
+        undefined;
```

Comment:
(Optional) If possible, maybe better? :D

```suggestion
    return config[`one-import-${ext}` as keyof typeof config]?.['input_path'];
```